src/test/java/com/exasol/adapter/document/files/extension/installer/GoInstaller.java

```diff
@@ -5,24 +5,30 @@
 import java.util.List;
 import java.util.logging.Logger;
 
+import com.exasol.adapter.document.files.extension.ExtensionTestConfig;
 import com.exasol.adapter.document.files.extension.OsCheck;
 import com.exasol.adapter.document.files.extension.process.SimpleProcess;
 
 class GoInstaller implements ExtensionManagerInstaller {
```

Comment:
Proposal
* InstallerFromLocalFolder
* InstallerFromGitHub

Reply:
I will rename this in a separate PR :)